### ISSUE TITLE
Table: Cell Type - Auto

### ISSUE THREAD

**Issue by drew08t** [grafana/grafana#94764]

**Comment by drew08t** [comment]
IMO, we should only enable hover expansion when wrap and cell inspect are disabled. What do you think @staton-hyse11 ?

**Comment by Develer** [comment]
> IMO, we should only enable hover expansion when wrap and cell inspect are disabled. What do you think @staton-hyse11 ?

Just for context, right now it can (randomly) extend to the right or to the bottom:

<img width="687" alt="Image" src="https://github.com/user-attachments/assets/61d4c5bc-ec3a-4b02-89ce-bf624a17e92c" />
<img width="692" alt="Image" src="https://github.com/user-attachments/assets/6641f54d-4e4b-4c6c-aea9-d94ab01009d9" />

Also, it continues to extend even if the text wrapper option is enabled, which doesn't make any sense.

cc @staton-hyse11

**Comment by staton-hyse11** [comment]
I agree with @drew08t. I would limit it to only expand down on hover. Expanding it to the right does not allow all content to be visible to the user.

**Comment by Develer** [comment]
@staton-hyse11 I have a question about the Cell value inspect option. Currently, we have the “Inspect cell” option in the context menu. However, the context menu is accessible on cell hover, and when a cell is hovered over, it automatically expands.

How should we handle both the context menu and the cell expansion behavior on hover?

cc @drew08t

**Comment by staton-hyse11** [comment]
It was discussed in our daily to revert this functionality to the old table behavior. This will maintain parity between new and old tables while a better approach to context menus is decided on in the future.
